Makefile

```diff
@@ -107,3 +107,9 @@ clean:
 	go clean -cache
 	rm -fr build/*
 	rm -fr contracts/compiled/*
+
+.PHONY: mocks
+mocks: ssz_types
+	@for p in $(shell grep -rl '//go:generate mockery' --exclude-dir=vendor | xargs -n1 dirname | sort -u); do \
+		go generate -run "mockery" $$p; \
+	done
```

Comment:
You can remove `go:generate` usage and use explicit make targets to avoid such strange constructions.

Reply:
Ideally, all the customization for what to generate mocks for should now be in `.mockery.yaml`. Via `include-interface-regex` or whatever.